eessi/testsuite/tests/apps/lammps/lammps.py

```diff
@@ -0,0 +1,102 @@
+"""
+This module tests the binary 'lmp' in available modules containing substring 'LAMMPS'.
+The tests come from the lammps github repository (https://github.com/lammps/lammps/)
+"""
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+from eessi.testsuite import hooks, utils
+from eessi.testsuite.constants import *  # noqa
+
+@rfm.simple_test
+class EESSI_LAMMPS(rfm.RunOnlyRegressionTest):
+    scale = parameter(SCALES.keys())
+    valid_prog_environs = ['default']
+    valid_systems = ['*']
+    time_limit = '30m'
+    tags = {TAGS['CI']}
+    device_type = parameter([DEVICE_TYPES[CPU], DEVICE_TYPES[GPU]])
+
+    # Parameterize over all modules that start with LAMMPS
+    module_name = parameter(utils.find_modules('LAMMPS'))
+    executable = 'lmp -in in.lj'
+
+    # Set sanity step
+    # TODO adjust for LAMMPS
+    @deferrable
+    def assert_tf_config_ranks(self):
```

Comment:
These are still the sanity parameters from the TensorFlow test need to be updated